permissions/plugin-portshift-scanner.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "portshift-scanner"
```

Comment:
This should match the artifactId from your pom.xml (https://github.com/jenkinsci/portshift-ci-plugin/blob/master/pom.xml#L11). Also, the name of the file should match the artifactId as well. So, please rename the file to plugin-portshift-ci.yml.